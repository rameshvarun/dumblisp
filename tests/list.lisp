(set a (list 2 3 4 "hello"))
(print (head a))
(print (len (tail a)))

(set b (cons 10 a))
(print (nth 0 b))
(print (nth 4 b))