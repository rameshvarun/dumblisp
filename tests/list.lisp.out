2
3
10
hello
